molecule/default/molecule.yml

```
@@ -0,0 +1,53 @@
+---
+dependency:
+  name: galaxy
+  enabled: true
+driver:
+  name: podman
```

Please use `containers` driver. It will pick the right one (`docker` or `podman`) based on your system configuration.

```suggestion
  name: containers
```

---

Yes. The switch to the `containers` driver makes sense for better flexibility between `Docker` and `Podman`.